styleguide.config.js

```diff
@@ -112,7 +129,7 @@ module.exports = {
                 overflow: "auto",
 
                 // Allow the preview content to grow to the full size, using
-                // absolute-positioning, and left/right/top/bottom all set to 0.
+                // absolute--positioning, and left/right/top/bottom all set to 0.
```

Comment:
Er...I didn't type anything here, so this change is a bit confusing. I'll try removing it.